src/libpgmoneta/prometheus.c

```diff
@@ -947,6 +1004,163 @@ general_information(int client_fd)
    }
    data = pgmoneta_append(data, "\n");
 
+   data = pgmoneta_append(data, "#HELP pgmoneta_server_timeline The current timeline a server is on\n");
+   data = pgmoneta_append(data, "#TYPE pgmoneta_server_timeline\n");
+   for (int i = 0; i < config->number_of_servers; i++)
+   {
+      data = pgmoneta_append(data, "pgmoneta_server_timeline{");
+
+      data = pgmoneta_append(data, "name=\"");
+      data = pgmoneta_append(data, config->servers[i].name);
+      data = pgmoneta_append(data, "\"} ");
+
+      data = pgmoneta_append_int(data, config->servers[i].cur_timeline);
+
+      data = pgmoneta_append(data, "\n");
+   }
+   data = pgmoneta_append(data, "\n");
+
+   data = pgmoneta_append(data, "#HELP pgmoneta_server_parent_tli The parent timeline of a timeline on a server\n");
+   data = pgmoneta_append(data, "#TYPE pgmoneta_server_parent_tli\n");
```

Comment:
Missing `counter`

Reply:
I didn't know there's a counter type, still a lot to learn I guess.

Reply:
Though as for parent TLI I think it's not garanteed to be increase monotonically, which is its main difference between `guage`. You can always restore from an way earlier tli.

Reply:
True, use `gauge`